app/models/article.rb

```diff
@@ -273,6 +273,14 @@ def self.trigger_index(record, remove)
     end
   end
 
+  def contrived_description
```

Comment:
can we call this `artificial_description` ? I had to google what `contrived` means 😅

#nonnativespeakersproblems

Reply:
Changed to `processed_description`

Reply:
thanks!